gatsby-config.js

```diff
@@ -431,18 +431,10 @@ module.exports = {
                         title: "Overview",
                         path: "/documentation/profile"
                     },
-                    {
-                        title: "Migration",
-                        path: "documentation/profile/migration"
-                    },
```

Comment:
The other question is - do we want to expand migration to include both iOS and Android? I'm not sure about this one haha.

Reply:
probably not because android 1.x/2.x apis are the same. also this will be covered only in the previous version section